.github/workflows/marker-report.yml

```diff
@@ -43,3 +68,54 @@ jobs:
         run: |
           . ./.venv/bin/activate
           python -m pytest tests/aws/
+
+        # makes use of the marker report plugin localstack.testing.pytest.marker_report
+      - name: Generate marker report
```

Comment:
Seems a bit redundant that we're doing this twice now (see `Collect marker report` above). Maybe only execute `Collect marker report` if `createIssue` is false could make sense? WDYT?

Reply:
This is something we can definitely think about. I was wondering what if we need an updated version of markers report in tinybird and also update/create a gh issue, the suggested implementation would mean that we need to run the workflow twice? We could maybe use two input params here, one to create github issue that we already have `createIssue` and another to collect the marker report in tinybird, what do you suggest?

Reply:
Generally the tinybird upload is meant to be running constantly (every commit on master) so we have a history to track, while the issue report is a manual (or potentially scheduled) run. So I think generally defaulting to tinybird upload, except when intentionally running it to create an issue would be fine